sharedbuffers/mapped_struct.py

```diff
@@ -1001,6 +1001,166 @@ def __str__(self):
     setattr(proxied_list, '__eq__', getattr(proxied_list, '_eq'))
     setattr(proxied_list, '__ne__', getattr(proxied_list, '_ne'))
 
+@cython.cclass
+class proxied_frozenset(object):
```

Comment:
Shouldn't you inherit from mapped_frozenset?

Reply:
None of the proxy classes inherit from the mapped ones, so I didn't see the point in doing that.

Reply:
It was an idea, since I expect you'll reuse a lot of the machinery there. But whatever.